test/fixtures/annotations.json

```diff
@@ -231,6 +231,16 @@
       "meta": {
         "classes": [
           "warning"
+        ],
+        "links": [
+          {
+            "element": "link",
+            "meta": {},
+            "attributes": {
+              "relation": "refract-not-supported"
```

Comment:
In using links, the plan was to use `origin` as the link relation and then a link to the origin of the issue.

```js
{
  element: 'link',
  attributes: {
    relation: 'origin',
    href: 'http://docs.apiary.io/parser/#refract-not-supported'
  }
}
```

Or something like that.